apply-ParagraphStandards.py

```diff
@@ -0,0 +1,149 @@
+import glob
+import json
+import yaml
+import getopt, sys
+from pprint import pprint
+from dateutil import parser
+
+def usage():
+    print ('Usage:')
+    print ("\t -m                               : modify source files")
+    print ("\t -t:{test}, or --test {test}      : run specific tests, or All (default)")
+    print ("\t -h, or --help                    : display help")
+    print ("\t Test list:")
+    print ("\t\t All")
+    print ("\t\t MDBlockGeneration")
+    print ("\t\t CodeBlockFix")
+
+try:
+    opts, args = getopt.getopt(sys.argv[1:], "-t:hm", ["help", "test=", "--modify"])
+except getopt.GetoptError as err:
+    # print help information and exit:
+    print(err) # will print something like "option -a not recognized"
+    usage()
+    sys.exit(2)
+modify = False
+tests = "All"
+for o, a in opts:
```

Comment:
Let's add another blank line above the `for`, for readability. Also, please use better variable names. At a minimum, please use `option` instead of `o` since `o` on its own is one of those letters that can be confused for a different character.